src/components/molecules/Footer/Footer.scss

```diff
@@ -7,14 +7,16 @@
 
   border-top: 1px solid rgba(255, 255, 255, 0.2);
   position: fixed;
-  height: $footer-height;
   bottom: 0;
   left: 0;
   z-index: z(footer);
   width: 100%;
   background-color: $black;
   padding-left: 30px;
   padding-right: 30px;
+  font-size: 0.7rem;
+  height: 30px;
```

Comment:
[TECH-DEBT] Removing the `$footer-height` variable from here (rather than changing it's value) has introduced visual styling bugs, as per https://github.com/sparkletown/internal-sparkle-issues/issues/233#issuecomment-776502846